src/functions/neptune/neptuneCaller.js

```diff
@@ -0,0 +1,30 @@
+var axios = require("axios");
+var uuid = require("uuid");
+var dayjs = require("dayjs");
+
+module.exports = async function (payload) {
+  var data = {
+    version: 1,
+    message_id: process.env.NEPTUNE_ENV + "-" + uuid.v4(),
+    timestamp: dayjs().unix(),
+  };
+  data = JSON.stringify({ ...data, ...payload });
+  console.log("chekc here data:::>", data);
+  var config = {
+    method: "post",
+    url: `${process.env.NEPTUNE_ENDPOINT}/${process.env.NEPTUNE_ENV}/events`,
```

Comment:
@Subbu-betalectic Can't we use neptune sdk here? https://www.npmjs.com/package/@teurons/neptune-nodejs

Reply:
Now using the neptune  sdk